src/Design/03_InitialPoC_trans_ffi.purs

```diff
@@ -0,0 +1,18 @@
+module FFI.PSPS.Design.InitialPoC.Trans where
+
+import PSPS (App, Foreign, Literal, Prim'Int, kind Expr, kind FFIName)
+import Type.Data.Peano (class ProductInt, class SumInt)
+
+class FFI_PSPS_Design_InitialPoC_Trans_Eval (expr :: Expr) (result :: Expr) | expr -> result
+
+foreign import data Foreign'intAdd :: FFIName
+
+instance iFFI_PSPS_Design_InitialPoC_Trans_Eval'intAdd'intAddE'App ::
+  SumInt a b c =>
```

Comment:
From here, as we've unpacked the integers from the literals, we just do whatever operation we want on the integers. Here, as we're adding, we use `SumInt` from `purescript-typelevel-peano`, which adds two integers.